be/src/storage/primary_index.cpp

```diff
@@ -1184,14 +1184,13 @@ Status PrimaryIndex::_do_load(Tablet* tablet) {
     _set_schema(pkey_schema);
 
     // load persistent index if enable persistent index meta
-    size_t fix_size = PrimaryKeyEncoder::get_encoded_fixed_size(_pk_schema);
 
-    if (tablet->get_enable_persistent_index() && (fix_size <= 128)) {
```

Comment:
is this a new function change to open the pk index limitation, not related to the bug fix?

Reply:
It is a historical problem, no need for now.